src/ToDo.UI/Styles/NavigationView.xaml

```diff
@@ -196,9 +223,9 @@
 										<Setter Target="PaneTransform.TranslateX" Value="0" />
 									</VisualState.Setters>
 									<Storyboard>
-										<xamarin:ObjectAnimationUsingKeyFrames Storyboard.TargetProperty="Clip" Storyboard.TargetName="PaneRoot">
+										<!--<xamarin:ObjectAnimationUsingKeyFrames Storyboard.TargetProperty="Clip" Storyboard.TargetName="PaneRoot">
 											<DiscreteObjectKeyFrame KeyTime="0:0:0" Value="{x:Null}" />
```

Comment:
Maybe a little comment on why these parts are commented for now please ;)

Reply:
Commented code removed